tbtadm/controller.cpp

```diff
@@ -399,13 +399,11 @@ void tbtadm::Controller::peers()
             continue;
         }
 
-        chdir(dir.path());
```

Comment:
Why this is better?
This why you have to do the path concatenating twice (for each file).

Reply:
I feel that string operations are less expensive then chdir(), it also does not work for std::filesystem